tests/openvino/native/quantization/test_ptq_params.py

```diff
@@ -13,28 +13,28 @@
 
 import pytest
 
-from nncf.common.utils.backend import get_backend
+from nncf.common.utils.backend import BackendType
 from nncf.common.graph.patterns.manager import PatternsManager
 from nncf.common.graph.patterns import GraphPattern
-from nncf.common.hardware.config import HW_CONFIG_TYPE_TARGET_DEVICE_MAP
 from nncf.parameters import TargetDevice
-from nncf.quantization.algorithms.definitions import RangeType
+from nncf.common.hardware.config import HW_CONFIG_TYPE_TARGET_DEVICE_MAP
 from nncf.quantization.algorithms.post_training.algorithm import PostTrainingQuantization
 from nncf.quantization.algorithms.post_training.algorithm import PostTrainingQuantizationParameters
-from nncf.quantization.algorithms.min_max.algorithm import MinMaxQuantization
 from nncf.experimental.openvino_native.graph.nncf_graph_builder import GraphConverter
 from nncf.experimental.openvino_native.quantization.algorithms.min_max.openvino_backend import OVMinMaxAlgoBackend
 from nncf.experimental.openvino_native.statistics.collectors import OVMeanMinMaxStatisticCollector
 from nncf.experimental.openvino_native.statistics.collectors import OVMinMaxStatisticCollector
 
-from tests.openvino.native.models import OVReferenceModel
 from tests.openvino.native.models import LinearModel
 from tests.openvino.native.models import DepthwiseConvModel
+from tests.post_training.test_ptq_params import TemplateTestPTQParams
 
 
-def get_patterns_setup(model: OVReferenceModel, device: TargetDevice) -> GraphPattern:
-    backend = get_backend(model)
-    return PatternsManager.get_full_pattern_graph(backend, device)
+def get_patterns_setup() -> GraphPattern:
+    backend = BackendType.OPENVINO
+    device = TargetDevice.ANY
+    patterns_manager = PatternsManager()
+    return patterns_manager.get_full_pattern_graph(backend, device)
```

Comment:
Since the method is static, we can use it without the `PatternsManager` instance.

Reply:
Done, thank you